cookbook/case_studies/ml_training/house_price_prediction/house_price_predictor.py

```diff
@@ -143,41 +172,44 @@ def split_data(
         ),
     )
 
-
-# %%
-# Task: Generating & Splitting the Data
-# ======================================
-# Call the previously defined helper functions to generate and split the data. Finally, return the DataFrame objects.
+# Now, let's create a NamedTuple that maps variable names (train_data) to data type (DataFrame).
+# This NamedTuple indicates that each variable name would store a DataFrame (train data, validation data, and test data respectively). 
```

Comment:
I think you are missing %%

```suggestion
# Let's create a ``NamedTuple`` that maps variable names to their respective data type.
```

Reply:
I actually thought this isn't a prominent part, so it could be a comment, rather than a heading. What do you think? @samhita-alla